src/components/LoanCards/LoanMatchingText.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+	<span class="tw-flex tw-items-center">
+		<h4 class="tw-inline-block ">
+			<span
+				class="
+				tw-mr-1
+				tw-p-1
+				tw-h-3
+				tw-rounded-full
+				tw-bg-brand-100
+				tw-border-2
+				tw-border-brand
```

Comment:
I think you may need an explicit width on this too that matches the height to ensure it's always a circle. Both the width and the height seem like they should be a bit smaller based on the screenshot.

As for layout, you might consider wrapping the icon in it's own span to gain more control over it as well if need be.

Reply:
I made some adjustments in my latest commit.